storefront/test/system/workarea/storefront/categories_system_test.rb

```diff
@@ -175,7 +177,8 @@ def test_filtering_products_and_sorting
         visit storefront.category_path(category)
 
         Capybara.match = :first
-        click_link '$10.00 - $19.99 (1)'
+        price_range = "#{currency}10.00 - #{currency}19.99"
```

Comment:
not all currencies have a prefix symbol, no?

Reply:
maybe I'm wrong, but in any case, you're doing this in a lot of places, I'd recommend creating a factory method for this.

Reply:
might be best to find a way to use `number_to_currency` so that behavior of generating this string matches what happens on the storefront

Reply:
I didn't think `number_to_currency` was available in tests...